imports/plugins/included/shipping-shippo/server/methods/shippo.js

```diff
@@ -349,6 +372,59 @@ export const methods = {
 
     return updatingResult;
   },
+  /**
+   * Formats and sends a cart address to Shippo to for validation
+   * @param {String} address - The address to validate
+   * @return {Object} validateAddress - The validation result from Shippo
+   */
+  "shippo/addressValidation"(address) {
+    check(address, Object);
+
+    const apiKey = getApiKey(Reaction.getShopId());
+
+    if (!apiKey) {
+      const msg = "No Shippo API key was found.";
+      Logger.fatal(msg);
+      throw new Meteor.Error("shippo-api-error", msg);
+    }
+
+    // get the buyer's email
+    const buyer = Accounts.findOne({ _id: this.userId }, { fields: { emails: 1 } });
+
+    if (!address) {
+      const msg = "The 'shipping' property of this cart is either missing or incomplete.";
+      Logger.error(msg);
+      throw new Meteor.Error("address-validation-error", msg);
+    }
+
+    const shop = Shops.findOne({ _id: Reaction.getShopId() }, { fields: { emails: 1 } });
+
+    // TODO take a more elegant approach to guest checkout -> no email address
+    let email = shop.emails[0].address || "noreply@localhost";
```

Comment:
I feel like there is a way to check for the `default` email rather just hardcoding the `0` index

Reply:
@zenweasel Just found `Reaction.getShopEmail()`, which does the exact same things, but will be better to use here.